src/Componenets/Form/Form.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import Signin from '../Signin/Signin.js';
-import Signup from '../Signup/Signup.js';
+import React from "react";
+import Signin from "../Signin/Signin.js";
+import Signup from "../Signup/Signup.js";
 import "bootstrap/dist/css/bootstrap.css";
 import "./Form.css";
-const Form = (props) => {
```

Comment:
Hey @Ajayjam97, why you are using props only for changing the active tab. I think we can do that without the involvement of props or without the involvement of react. 
The way in which you did this it will not work. Please have a look at it and try to correct it. :+1:

Reply:
@lalitmee He didnt add props, I did. The props is passing 2 event handlers and a part of state for switching and rendering data between Signup/Signin page and adding/removing active class in the tabs. Its a necessity.

Reply:
but actually, it's not working. Earlier the tabs were switching but now they are not.

Reply:
That's weird, it seems to be working for me.

Reply:
@lalitmee  Yes, I only added the border color to the input fields.